src/contexts/XplaWalletContext.tsx

```diff
@@ -1,38 +1,20 @@
-import { NetworkInfo, WalletProvider } from "@xpla/wallet-provider";
-import { ReactChildren } from "react";
+import { CHAIN_ID_XPLA } from "@xlabs-libs/wallet-aggregator-core";
+import { useWallet } from "@xlabs-libs/wallet-aggregator-react";
+import { getWallets, XplaWallet } from "@xlabs-libs/wallet-aggregator-xpla";
+import { ConnectType } from "@xpla/wallet-provider";
 
-const testnet: NetworkInfo = {
-  name: "testnet",
-  chainID: "cube_47-5",
-  lcd: "https://cube-lcd.xpla.dev",
-  walletconnectID: 0,
-};
+export const getXplaWallets = async () => {
+  let xplaWallets: XplaWallet[] = [];
 
-const mainnet: NetworkInfo = {
-  name: "mainnet",
-  chainID: "dimension_37-1",
-  lcd: "https://dimension-lcd.xpla.dev",
-  walletconnectID: 1,
-};
+  try {
+    xplaWallets = await getWallets([ConnectType.READONLY]);
+  } catch (err) {
+    console.error("Failed to init terra chain wallets. Error:", err);
```

Comment:
Is the `init terra chain` message correct on this file?

Reply:
That's right, I slipped there!